From 34a195ea6870c2f7b50ffe1d60a924ebf15b1542 Mon Sep 17 00:00:00 2001
From: Allan Stephens <allan.stephens@windriver.com>
Date: Fri, 19 Nov 2010 14:20:39 -0500
Subject: [PATCH 26/32] tipc: Eliminate an unused symbolic constant in link code

commit 5082636dff42896673e73e850e4650d1ea6b1e68 from
git://tipc.cslab.ericsson.net/pub/git/people/allan/tipc.git

Removes a symbol that is not referenced anywhere by TIPC's link code.

Signed-off-by: Allan Stephens <allan.stephens@windriver.com>
Integrated-by: Howard Xu <hao.xu@windriver.com>
---
 net/tipc/tipc_link.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/net/tipc/tipc_link.c b/net/tipc/tipc_link.c
index a64816c..85f06e8 100644
--- a/net/tipc/tipc_link.c
+++ b/net/tipc/tipc_link.c
@@ -56,12 +56,6 @@
 #define INVALID_SESSION 0x10000
 
 /*
- * Limit for deferred reception queue:
- */
-
-#define DEF_QUEUE_LIMIT 256u
-
-/*
  * Link state events:
  */
 
-- 
1.7.0

